tools/docker/sles.docker

```diff
@@ -19,7 +19,16 @@ RUN zypper --gpg-auto-import-keys install -y \
     gcc-c++ \
     gdb \
     git \
-    python3-pip
+    python3-pip \
+    rpm-build
+
+#addition of repos for packages
+RUN OPENSUSE_REPO=https://download.opensuse.org/repositories && \
+    zypper addrepo ${OPENSUSE_REPO}/science/SLE_15_SP4/science.repo && \
+    zypper addrepo ${OPENSUSE_REPO}/server:/http/15.4/server:http.repo && \
+    zypper addrepo ${OPENSUSE_REPO}/devel:/languages:/perl/SLE_15_SP4/devel:languages:perl.repo && \
+    zypper addrepo ${OPENSUSE_REPO}/devel:/tools/15.4/devel:tools.repo && \
+    zypper addrepo ${OPENSUSE_REPO}/multimedia:/libs/15.4/multimedia:libs.repo
```

Comment:
Where in the rocm documentation do we have these repos documented?

Reply:
DevOps team's SLES docker needed to resolve our ./install_prereqs.sh file

Reply:
I removed the lines that are not needed